dashboard/app/bisect_test.go

```diff
@@ -188,11 +188,13 @@ For information about bisection process see: https://goo.gl/tpsmEJ#bisection
 
```

Comment:
This test doesn't seem to actually test `Recipients`.  Can you make sure that `Recipients` gets properly tested?  Likewise in the other test files too.